submariner/cable-driver-protocols-enhancement.md

```diff
@@ -0,0 +1,76 @@
+# Cable Driver Enhancement - IP-in-IP
+
+## Summary
+
+Submariner today supports three cable drivers: VXLAN, IPSec (tunnel mode) and WireGuard. The
+goal of this proposal is to outline additional enhancements needed for the Submariner Cable Driver
+to support of IP-in-IP for unencrypted connections between clusters or for use as an overlay with
+IPSec in transport mode for encrypted connections between clusters.
+
+For IP in IP, the benefits fall into the following categories:
+
```

Comment:
It makes sense also to note in the proposal if it's possible or not to run SM IPIP traffic with GW as LB service.